### PR TITLE
feat(devtools): create demo application that uses standalone APIs/components/directives/pipes

### PR DESCRIPTION
**Pull request by AleksanderBodurri**
The existing DevTools demo app that is used for developing on DevTools is exclusively an NgModule application. This commit creates a copy of the old demo app but with no NgModules and only standalone APIs/Components/Directives/Pipes.

In the future this will lets us run tests against applications that both use and do not use NgModules. In the present it serves as a convenient playground for developing new debug APIs in an NgModule-less application.

### CODE REVIEW

File: devtools/projects/demo-standalone/src/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>AngularDevtools</title>
+    <base href="/" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="assets/icon16.png" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+
+    <!-- <script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.0.3/custom-elements-es5-adapter.js"></script> -->
```
Comment (josephperrott) [inline devtools/projects/demo-standalone/src/index.html:11]: Does this still need to be here? Can we remove it?

**Review by josephperrott — APPROVED** [review]
LGTM

File: devtools/projects/demo-standalone/src/index.html (hunk 1 above)
Comment (AleksanderBodurri) [reply devtools/projects/demo-standalone/src/index.html]: Thanks for catching that, it does not. Removed 😄

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit dbadfea67f117f559d6387176b3076d6f055fdc6.

**Pull request closed without merging**